marklogic-langchain4j/src/main/java/com/marklogic/langchain4j/classifier/TextClassifier.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright © 2025 MarkLogic Corporation. All Rights Reserved.
+ */
+package com.marklogic.langchain4j.classifier;
+
+import com.smartlogic.classificationserver.client.*;
+import com.smartlogic.cloud.CloudException;
+import com.smartlogic.cloud.Token;
+import com.smartlogic.cloud.TokenFetcher;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+public class TextClassifier {
+
+    private static final String THRESHOLD = "20";
+
+    private final ClassificationClient classificationClient;
+    private Token token;
+    private final TokenFetcher tokenFetcher;
+
+    public TextClassifier(String host, String protocol, String port, String endpoint, String apikey, String tokenEndpoint) {
```

Comment:
Sonar may complain about 6 constructor args too. If so, just ignore it for now, we can add a Builder or something later.